### ISSUE TITLE
housemates should always be included and should be at the top of the list

### ISSUE THREAD

**Issue by KenStanley** [DaleStan/VoterMate#24]
This ought to show up just because of the lat, lon distance being zero, but that did not seem to always be the case. I will add specific examples soon and will assign this to myself until I have something specific.

**Comment by KenStanley** [comment]
I strongly suspect that this is a lat,lon issue. It should stay assigned to me until I get the lat,lon thing fixed and then we'll see.

**Comment by KenStanley** [comment]
I believe that I have fixed the lat, lon for one precinct, MAN 4 - E 
I did this the brute force way, calling google to look up the lat,lon for every address (though not separately for apartments. 
As a separate matter I will see if I can salvage lat lon from other sources or if I need to look them up for all precincts that we will canvass. Google allows hundreds (maybe thousands) of lat,lon lookups for free, but it gets pricey at some point.

**Comment by DaleStan** [comment]
The Google sheet still has several Sackman St houses at 40.77302, -82.54157, on both voterDB and canvass, though I don't need particularly accurate information unless this ends up on my plate.

**Comment by KenStanley** [comment]
That was just a stupid mistake on my part. I have now fixed that and when I drove down Sackman Street this morning, it pick up all 18 houses that were on the canvass between 592 and 634. This is the set that I chose to check. 
I also saw housemates showing up. I didn't check all of the housemates, but those that I did check were there, except that Duquan (late 20s) was not on Cassandra(50ish)'s list, though she was on his. This is backwards of what I would expect as older people tend to be more reliable voters. I will move this to an issue about duibe checking our lists to make sure that I have the right people on the list.
